lib/tariff_synchronizer/file_service.rb

```diff
@@ -67,6 +55,15 @@ def download_content(url)
 
       private
 
+      def store_file_in_production(path, body)
```

Comment:
👌 ok, makes sense, I always try to use conditionals as minimum as possible, but maybe in this case is simple enough to add one.